src/Sentry/Internal/ReplaySession.cs

```diff
@@ -0,0 +1,33 @@
+#if __ANDROID__
+using Sentry.Android.Extensions;
+#endif
+
+namespace Sentry.Internal;
+
+internal interface IReplaySession
```

Comment:
question: Is "(I)ReplaySession" the right name?

To be fair, I'm not too familiar with Session-Replay yet. And also, it's an `internal` API.
But I am uncertain whether I got confused here a little.
Is this indeed representing a "Replay-Session" ... or is this more of an "Replay-Session-Accessor" ... I apparently think about the `IHttpContextAccessor`, which the ASP.NET Core team admitted isn't the best pattern either.

TL;DR; not sure if there would be a better name?
Or is this name matching native concepts (which which I'm admittedly not too familiar with yet)

Reply:
I just made this name up. All it does is get the ReplayId (from the Java SDK in the case of Android applications and from some mock in the case of unit tests). I figured when it was accessed it would read like:
```csharp
var replayId = ReplaySession.ActiveReplayId
```

So it gets the `ActiveReplayId` for the "current replay session". If there is not replay session then the `ActiveReplayId` will be null. Seemed like it would read well.